doc/configuration.rst

```diff
@@ -143,6 +143,56 @@ You can specify the following options in module configuration.
         position = 'center'
     }
 
+.. note::
+    New in version 3.16
+
+You can specify the options in module or py3status configuration section.
+
+The following options will work on ``i3``.
+
+``align``: Specify how modules should be aligned when the ``min_width``
+is not reached. Either ``left`` (default), ``center``, or ``right``.
+``markup``: Specify how modules should be parsed.
+``min_width``: Specify a minimum width of pixels for modules.
+``separator``: Specify a separator boolean for modules.
+``separator_block_width``: Specify a separator block width for modules.
+
+The following options will work on ``i3-gaps``.
+
+``background``: Specify a background color for modules.
+``border``: Specify a border color for modules.
+``border_bottom``: Specify a border width for modules
+``border_left``: Specify a border width for modules.
+``border_right``: Specify a border width for modules.
+``border_top``: Specify a border width for modules.
+
+The following options will work on ``py3status``.
```

Comment:
you mean, on py3status modules right? plz rephrase

Reply:
It should work on all modules, but should be only used only on py3status modules because we use multiple composites.

I said all modules because one composite is just one composite. `min_length` and `position` should have no issue dealing with a single composite.

Reply:
I meant it will only work with `py3status`... Not part of i3bar or i3bar-gaps.

Reply:
This is correct as-is because..  Custom `min_length` and `position` to deal with multiple composites. Single composite included. `py3status` deals with it, not `i3bar` or `i3bar_gaps`.